### PR TITLE
Feature simplify configurations

### PR DESCRIPTION
**Pull request by Robert108**
This PR removes most of the remaining non-working configuration options that are left from Chronos/Maestro. 

- test.conf is gone and required fields in the object .opro files that were not used are no longer parsed to avoid misdirection.
- Old .traj, scenario and geofence files are removed as well. 
- File validation is moved to launch python files to remove first time start error. 
- Kept example scenario files and and configuration in params.yaml
- Create a working .astazero/ATOS directory if none exists on start up. 
- Added quickstart guide to docs.

### CODE REVIEW

File: launch/launch_experimental.py
Comment (victorjarlow) [inline launch/launch_experimental.py]:
```suggestion
sys.path.insert(0, os.path.join( # Need to modify the sys.path since we launch from the ros2 installed path
```

File: docs/Installation/quickstart.md
Comment (victorjarlow) [inline docs/Installation/quickstart.md]:
```suggestion
* Download and build the client software _ISO\_objectDemo_ from the [isoObject repo](https://github.com/RI-SE/isoObject). Follow the build instructions and execute the binary ISO_objectDemo in the build folder. This client software creates a virtual test object which will respond to control messages from ATOS. This software can be used to verify the correctness of communication between server and client)
```

File: docs/Installation/quickstart.md
Comment (victorjarlow) [inline docs/Installation/quickstart.md]:
```suggestion
This section will walk you through the steps of starting the ATOS server for the first time. 
```

File: docs/Installation/quickstart.md
Comment (victorjarlow) [inline docs/Installation/quickstart.md]:
```suggestion
* Start ATOS by running either one of the docker commands (or with "ros2 launch ..." if you have installed it locally). In this guide, the docker compose command is used.```

File: docs/Installation/quickstart.md
Comment (victorjarlow) [inline docs/Installation/quickstart.md:29]:
```suggestion

Repeat the above steps for the self-signed WebSocket SSL certificate [https://localhost:9090](https://localhost:9090)
```

File: common/util.h
Comment (samuelthoren) [inline common/util.h]: Are these functions still used?

File: conf/objects/TestObject1.opro
Comment (samuelthoren) [inline conf/objects/TestObject1.opro]:
```suggestion
# The ID value should match the EgoModelId parameter value in the Catalogs/Vehicles/VehicleCatalog.xosc file 
```

File: conf/odr/README.md
Comment (samuelthoren) [inline conf/odr/README.md]:
```suggestion
This folder contains OpenDrive files of various test areas at AstaZero. 
```

Comment (samuelthoren) [reply conf/odr/README.md]: Wasn't areas the word to be used instead of tracks. I might be remembering it wrong though

File: docs/Installation/connected.gif
Comment (samuelthoren) [inline docs/Installation/connected.gif:1]: Nice visualisation! Could we maybe change the map to the more high resolution though?

File: docs/Installation/installation.md
Comment (samuelthoren) [inline docs/Installation/installation.md]:
```suggestion
ATOS comes with an installation script that automates the installation process. It is intended for use on Ubuntu 20.04 or 22.04, and has been tested on a fresh install of Ubuntu 20.04. The script will install ROS 2, ATOS dependencies, and ATOS itself. It will also create a workspace and build ATOS. The script can be executed using the following command:
```

File: docs/Installation/installation.md
Comment (samuelthoren) [inline docs/Installation/installation.md]:
```suggestion
When running ATOS for the first time and the no pre-existing .astazero/ATOS folder is found, ATOS will create some barebone configuration files which you can read more about in the configuration section [here](../Usage/How-to/configuration.md).
```

File: docs/Installation/quickstart.md
Comment (samuelthoren) [inline docs/Installation/quickstart.md]:
```suggestion
* Download and build the ISO_objectDemo binary from the [isoObject repo](https://github.com/RI-SE/isoObject). Follow the build instructions and execute the binary ISO_objectDemo in the build folder. (This object is not controllable but will respond to control messages from ATOS. It is used to test the connection between ATOS and the object.)
```

File: launch/launch_experimental.py
Comment (samuelthoren) [inline launch/launch_experimental.py:30]: I don't think the OSI adapter should be in the experimental launch file. We won't be able to stream to simulated objects (CARLA, etc) without it.

File: launch/launch_utils/launch_base.py
Comment (samuelthoren) [inline launch/launch_utils/launch_base.py]: The pointcloud publisher can maybe be moved to the graphical launch file?

File: modules/DirectControl/src/directcontrol.cpp
Comment (samuelthoren) [inline modules/DirectControl/src/directcontrol.cpp:217]: Is this something that needs to be done before we merge?

File: plugins/foxglove/Map, 3D and control layout.json
Comment (samuelthoren) [inline plugins/foxglove/Map, 3D and control layout.json]: Should this be here, only we can access it

**Review by samuelthoren — COMMENTED** [review]
Very nice! It's good to add an easy way to get started with ATOS. I have added some commits for writing the scenario info to the journals when a test is done. I will do some more testing before I'll approve it and I will see if I can find some more things to change/remove.

File: modules/JournalControl/src/journalcontrol.cpp
Comment (samuelthoren) [inline modules/JournalControl/src/journalcontrol.cpp:53]: Is there a default value for this? If I don't set a name in params.yaml, no file is created

Comment (samuelthoren) [reply modules/JournalControl/src/journalcontrol.cpp:53]: Also I think we will have to decide if we want all information in one file, or if we should split it up between several files. What I can see now with this function is that it creates two separate files. I have added in another commit that we print the scenario info with the trajectories, so we can either move that function here or we can maybe remove this function altogether

**Review by victorjarlow — APPROVED** [review]

File: modules/DirectControl/src/directcontrol.cpp
Comment (victorjarlow) [reply modules/DirectControl/src/directcontrol.cpp:217]: The code is not relevant anymore, can be implemented using equivalent ros2 features. Line was removed since datadictionary was deleted in this PR

File: launch/launch_experimental.py
Comment (samuelthoren) [reply launch/launch_experimental.py:30]: TO DO: Make a launch file (or solve in another way) where we can launch all modules that we used at the demo. This is currently not possible if we move the pointcloud publisher to graphical and OSI-adapter to experimental.

**Comment by samuelthoren** [comment]
Also, should we still keep rviz? It is still launching with `launch_graphical`, or can I remove it and from now on we only use Foxglove? There is also no mention of rviz in the documentation.

**Review by samuelthoren — APPROVED** [review]

**Pull request merged**
